lib/push.js

```diff
@@ -8,13 +8,16 @@ var P = require('./promise')
 var ERR_NO_PUSH_CALLBACK = 'No Push Callback'
 var ERR_DATA_BUT_NO_KEYS = 'Data payload present but missing key(s)'
 
-var LOG_OP_PUSH_TO_DEVICES = 'push.pushToDevices'
+var LOG_OP_PUSH_TO_DEVICES = 'push.sendPush'
```

Comment:
@eoger could you please not change this `op`, we have a Kibana filter in the works that uses that name